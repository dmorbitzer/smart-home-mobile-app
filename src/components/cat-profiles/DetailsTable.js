import { Text, HStack, VStack } from "@react-native-material/core";

export default function DetailsTable(props) {
  const date = new Date(props.data.cat.birthdate);
  const localGender =
    props.data.cat.gender.name === "male" ? "MÃ¤nnlich" : "Weiblich";
  return (
    <HStack>
      <VStack fill center spacing={4} divider>
        <Text variant="h6">Geschlecht</Text>
        <Text variant="h6">Rasse</Text>
        <Text variant="h6">Geburtstag</Text>
        <Text variant="h6">Gewicht</Text>
      </VStack>
      <VStack fill center spacing={4} divider>
        <Text variant="h6">{localGender}</Text>
        <Text variant="h6">{props.data.cat.race}</Text>
        <Text variant="h6">{date.toLocaleDateString("de-DE")}</Text>
        <Text variant="h6">{props.data.cat.weight} kg</Text>
      </VStack>
    </HStack>
  );
}
