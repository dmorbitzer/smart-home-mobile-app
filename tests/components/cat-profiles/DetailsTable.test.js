import React from "react";
import renderer from "react-test-renderer";

import DetailsTable from "../../../src/components/cat-profiles/DetailsTable";
jest.useFakeTimers();
const props = {
  data: {
    cat: {
      id: "/api/cats/245",
      name: "Bla",
      birthdate: "2023-04-14T00:00:00+02:00",
      gender: {
        name: "male",
      },
      race: "super",
      weight: 2,
    },
  },
};
test("renders DetailsTable correctly", () => {
  const tree = renderer.create(<DetailsTable data={props.data} />).toJSON();
  expect(tree).toMatchSnapshot();
});
