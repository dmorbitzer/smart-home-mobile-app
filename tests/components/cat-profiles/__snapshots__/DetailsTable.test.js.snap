// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DetailsTable correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": undefined,
        "borderBottomColor": undefined,
        "borderBottomEndRadius": undefined,
        "borderBottomLeftRadius": undefined,
        "borderBottomRightRadius": undefined,
        "borderBottomStartRadius": undefined,
        "borderBottomWidth": undefined,
        "borderColor": undefined,
        "borderEndColor": undefined,
        "borderEndWidth": undefined,
        "borderLeftColor": undefined,
        "borderLeftWidth": undefined,
        "borderRadius": undefined,
        "borderRightColor": undefined,
        "borderRightWidth": undefined,
        "borderStartColor": undefined,
        "borderStartWidth": undefined,
        "borderStyle": undefined,
        "borderTopColor": undefined,
        "borderTopEndRadius": undefined,
        "borderTopLeftRadius": undefined,
        "borderTopRightRadius": undefined,
        "borderTopStartRadius": undefined,
        "borderTopWidth": undefined,
        "borderWidth": undefined,
        "bottom": undefined,
        "end": undefined,
        "height": undefined,
        "left": undefined,
        "margin": undefined,
        "marginBottom": undefined,
        "marginEnd": undefined,
        "marginHorizontal": undefined,
        "marginLeft": undefined,
        "marginRight": undefined,
        "marginStart": undefined,
        "marginTop": undefined,
        "marginVertical": undefined,
        "maxHeight": undefined,
        "maxWidth": undefined,
        "minHeight": undefined,
        "minWidth": undefined,
        "opacity": undefined,
        "overflow": undefined,
        "padding": undefined,
        "paddingBottom": undefined,
        "paddingEnd": undefined,
        "paddingHorizontal": undefined,
        "paddingLeft": undefined,
        "paddingRight": undefined,
        "paddingStart": undefined,
        "paddingTop": undefined,
        "paddingVertical": undefined,
        "position": undefined,
        "right": undefined,
        "start": undefined,
        "top": undefined,
        "width": undefined,
        "zIndex": undefined,
      },
      [
        {
          "alignContent": undefined,
          "alignItems": undefined,
          "alignSelf": undefined,
          "flex": undefined,
          "flexBasis": undefined,
          "flexDirection": "row",
          "flexGrow": undefined,
          "flexShrink": undefined,
          "flexWrap": undefined,
          "justifyContent": undefined,
        },
        undefined,
      ],
    ]
  }
>
  <View
    style={
      [
        {
          "backgroundColor": undefined,
          "borderBottomColor": undefined,
          "borderBottomEndRadius": undefined,
          "borderBottomLeftRadius": undefined,
          "borderBottomRightRadius": undefined,
          "borderBottomStartRadius": undefined,
          "borderBottomWidth": undefined,
          "borderColor": undefined,
          "borderEndColor": undefined,
          "borderEndWidth": undefined,
          "borderLeftColor": undefined,
          "borderLeftWidth": undefined,
          "borderRadius": undefined,
          "borderRightColor": undefined,
          "borderRightWidth": undefined,
          "borderStartColor": undefined,
          "borderStartWidth": undefined,
          "borderStyle": undefined,
          "borderTopColor": undefined,
          "borderTopEndRadius": undefined,
          "borderTopLeftRadius": undefined,
          "borderTopRightRadius": undefined,
          "borderTopStartRadius": undefined,
          "borderTopWidth": undefined,
          "borderWidth": undefined,
          "bottom": undefined,
          "end": undefined,
          "height": undefined,
          "left": undefined,
          "margin": undefined,
          "marginBottom": undefined,
          "marginEnd": undefined,
          "marginHorizontal": undefined,
          "marginLeft": undefined,
          "marginRight": undefined,
          "marginStart": undefined,
          "marginTop": undefined,
          "marginVertical": undefined,
          "maxHeight": undefined,
          "maxWidth": undefined,
          "minHeight": undefined,
          "minWidth": undefined,
          "opacity": undefined,
          "overflow": undefined,
          "padding": undefined,
          "paddingBottom": undefined,
          "paddingEnd": undefined,
          "paddingHorizontal": undefined,
          "paddingLeft": undefined,
          "paddingRight": undefined,
          "paddingStart": undefined,
          "paddingTop": undefined,
          "paddingVertical": undefined,
          "position": undefined,
          "right": undefined,
          "start": undefined,
          "top": undefined,
          "width": undefined,
          "zIndex": undefined,
        },
        [
          {
            "alignContent": undefined,
            "alignItems": "center",
            "alignSelf": undefined,
            "flex": 1,
            "flexBasis": undefined,
            "flexDirection": "column",
            "flexGrow": undefined,
            "flexShrink": undefined,
            "flexWrap": undefined,
            "justifyContent": "center",
          },
          [
            undefined,
            [
              {
                "marginEnd": 0,
              },
              undefined,
            ],
          ],
        ],
      ]
    }
  >
    <Text
      style={
        [
          {
            "fontSize": 20,
            "fontWeight": "500",
            "letterSpacing": 0.15,
          },
          {
            "color": "#000000",
          },
          [
            undefined,
            [
              {
                "marginBottom": 4,
              },
              undefined,
            ],
          ],
        ]
      }
    >
      Geschlecht
    </Text>
    <View
      style={
        [
          {
            "backgroundColor": undefined,
            "borderBottomColor": undefined,
            "borderBottomEndRadius": undefined,
            "borderBottomLeftRadius": undefined,
            "borderBottomRightRadius": undefined,
            "borderBottomStartRadius": undefined,
            "borderBottomWidth": undefined,
            "borderColor": undefined,
            "borderEndColor": undefined,
            "borderEndWidth": undefined,
            "borderLeftColor": undefined,
            "borderLeftWidth": undefined,
            "borderRadius": undefined,
            "borderRightColor": undefined,
            "borderRightWidth": undefined,
            "borderStartColor": undefined,
            "borderStartWidth": undefined,
            "borderStyle": undefined,
            "borderTopColor": undefined,
            "borderTopEndRadius": undefined,
            "borderTopLeftRadius": undefined,
            "borderTopRightRadius": undefined,
            "borderTopStartRadius": undefined,
            "borderTopWidth": undefined,
            "borderWidth": undefined,
            "bottom": undefined,
            "end": undefined,
            "height": undefined,
            "left": undefined,
            "margin": undefined,
            "marginBottom": undefined,
            "marginEnd": undefined,
            "marginHorizontal": undefined,
            "marginLeft": undefined,
            "marginRight": undefined,
            "marginStart": undefined,
            "marginTop": undefined,
            "marginVertical": undefined,
            "maxHeight": undefined,
            "maxWidth": undefined,
            "minHeight": undefined,
            "minWidth": undefined,
            "opacity": undefined,
            "overflow": undefined,
            "padding": undefined,
            "paddingBottom": undefined,
            "paddingEnd": undefined,
            "paddingHorizontal": undefined,
            "paddingLeft": undefined,
            "paddingRight": undefined,
            "paddingStart": undefined,
            "paddingTop": undefined,
            "paddingVertical": undefined,
            "position": undefined,
            "right": undefined,
            "start": undefined,
            "top": undefined,
            "width": undefined,
            "zIndex": undefined,
          },
          [
            {
              "backgroundColor": "#E0E0E0",
              "height": 1,
            },
            [
              [
                undefined,
                undefined,
              ],
              [
                {
                  "marginBottom": 4,
                },
                undefined,
              ],
            ],
          ],
        ]
      }
    />
    <Text
      style={
        [
          {
            "fontSize": 20,
            "fontWeight": "500",
            "letterSpacing": 0.15,
          },
          {
            "color": "#000000",
          },
          [
            undefined,
            [
              {
                "marginBottom": 4,
              },
              undefined,
            ],
          ],
        ]
      }
    >
      Rasse
    </Text>
    <View
      style={
        [
          {
            "backgroundColor": undefined,
            "borderBottomColor": undefined,
            "borderBottomEndRadius": undefined,
            "borderBottomLeftRadius": undefined,
            "borderBottomRightRadius": undefined,
            "borderBottomStartRadius": undefined,
            "borderBottomWidth": undefined,
            "borderColor": undefined,
            "borderEndColor": undefined,
            "borderEndWidth": undefined,
            "borderLeftColor": undefined,
            "borderLeftWidth": undefined,
            "borderRadius": undefined,
            "borderRightColor": undefined,
            "borderRightWidth": undefined,
            "borderStartColor": undefined,
            "borderStartWidth": undefined,
            "borderStyle": undefined,
            "borderTopColor": undefined,
            "borderTopEndRadius": undefined,
            "borderTopLeftRadius": undefined,
            "borderTopRightRadius": undefined,
            "borderTopStartRadius": undefined,
            "borderTopWidth": undefined,
            "borderWidth": undefined,
            "bottom": undefined,
            "end": undefined,
            "height": undefined,
            "left": undefined,
            "margin": undefined,
            "marginBottom": undefined,
            "marginEnd": undefined,
            "marginHorizontal": undefined,
            "marginLeft": undefined,
            "marginRight": undefined,
            "marginStart": undefined,
            "marginTop": undefined,
            "marginVertical": undefined,
            "maxHeight": undefined,
            "maxWidth": undefined,
            "minHeight": undefined,
            "minWidth": undefined,
            "opacity": undefined,
            "overflow": undefined,
            "padding": undefined,
            "paddingBottom": undefined,
            "paddingEnd": undefined,
            "paddingHorizontal": undefined,
            "paddingLeft": undefined,
            "paddingRight": undefined,
            "paddingStart": undefined,
            "paddingTop": undefined,
            "paddingVertical": undefined,
            "position": undefined,
            "right": undefined,
            "start": undefined,
            "top": undefined,
            "width": undefined,
            "zIndex": undefined,
          },
          [
            {
              "backgroundColor": "#E0E0E0",
              "height": 1,
            },
            [
              [
                undefined,
                undefined,
              ],
              [
                {
                  "marginBottom": 4,
                },
                undefined,
              ],
            ],
          ],
        ]
      }
    />
    <Text
      style={
        [
          {
            "fontSize": 20,
            "fontWeight": "500",
            "letterSpacing": 0.15,
          },
          {
            "color": "#000000",
          },
          [
            undefined,
            [
              {
                "marginBottom": 4,
              },
              undefined,
            ],
          ],
        ]
      }
    >
      Geburtstag
    </Text>
    <View
      style={
        [
          {
            "backgroundColor": undefined,
            "borderBottomColor": undefined,
            "borderBottomEndRadius": undefined,
            "borderBottomLeftRadius": undefined,
            "borderBottomRightRadius": undefined,
            "borderBottomStartRadius": undefined,
            "borderBottomWidth": undefined,
            "borderColor": undefined,
            "borderEndColor": undefined,
            "borderEndWidth": undefined,
            "borderLeftColor": undefined,
            "borderLeftWidth": undefined,
            "borderRadius": undefined,
            "borderRightColor": undefined,
            "borderRightWidth": undefined,
            "borderStartColor": undefined,
            "borderStartWidth": undefined,
            "borderStyle": undefined,
            "borderTopColor": undefined,
            "borderTopEndRadius": undefined,
            "borderTopLeftRadius": undefined,
            "borderTopRightRadius": undefined,
            "borderTopStartRadius": undefined,
            "borderTopWidth": undefined,
            "borderWidth": undefined,
            "bottom": undefined,
            "end": undefined,
            "height": undefined,
            "left": undefined,
            "margin": undefined,
            "marginBottom": undefined,
            "marginEnd": undefined,
            "marginHorizontal": undefined,
            "marginLeft": undefined,
            "marginRight": undefined,
            "marginStart": undefined,
            "marginTop": undefined,
            "marginVertical": undefined,
            "maxHeight": undefined,
            "maxWidth": undefined,
            "minHeight": undefined,
            "minWidth": undefined,
            "opacity": undefined,
            "overflow": undefined,
            "padding": undefined,
            "paddingBottom": undefined,
            "paddingEnd": undefined,
            "paddingHorizontal": undefined,
            "paddingLeft": undefined,
            "paddingRight": undefined,
            "paddingStart": undefined,
            "paddingTop": undefined,
            "paddingVertical": undefined,
            "position": undefined,
            "right": undefined,
            "start": undefined,
            "top": undefined,
            "width": undefined,
            "zIndex": undefined,
          },
          [
            {
              "backgroundColor": "#E0E0E0",
              "height": 1,
            },
            [
              [
                undefined,
                undefined,
              ],
              [
                {
                  "marginBottom": 4,
                },
                undefined,
              ],
            ],
          ],
        ]
      }
    />
    <Text
      style={
        [
          {
            "fontSize": 20,
            "fontWeight": "500",
            "letterSpacing": 0.15,
          },
          {
            "color": "#000000",
          },
          [
            undefined,
            [
              null,
              undefined,
            ],
          ],
        ]
      }
    >
      Gewicht
    </Text>
  </View>
  <View
    style={
      [
        {
          "backgroundColor": undefined,
          "borderBottomColor": undefined,
          "borderBottomEndRadius": undefined,
          "borderBottomLeftRadius": undefined,
          "borderBottomRightRadius": undefined,
          "borderBottomStartRadius": undefined,
          "borderBottomWidth": undefined,
          "borderColor": undefined,
          "borderEndColor": undefined,
          "borderEndWidth": undefined,
          "borderLeftColor": undefined,
          "borderLeftWidth": undefined,
          "borderRadius": undefined,
          "borderRightColor": undefined,
          "borderRightWidth": undefined,
          "borderStartColor": undefined,
          "borderStartWidth": undefined,
          "borderStyle": undefined,
          "borderTopColor": undefined,
          "borderTopEndRadius": undefined,
          "borderTopLeftRadius": undefined,
          "borderTopRightRadius": undefined,
          "borderTopStartRadius": undefined,
          "borderTopWidth": undefined,
          "borderWidth": undefined,
          "bottom": undefined,
          "end": undefined,
          "height": undefined,
          "left": undefined,
          "margin": undefined,
          "marginBottom": undefined,
          "marginEnd": undefined,
          "marginHorizontal": undefined,
          "marginLeft": undefined,
          "marginRight": undefined,
          "marginStart": undefined,
          "marginTop": undefined,
          "marginVertical": undefined,
          "maxHeight": undefined,
          "maxWidth": undefined,
          "minHeight": undefined,
          "minWidth": undefined,
          "opacity": undefined,
          "overflow": undefined,
          "padding": undefined,
          "paddingBottom": undefined,
          "paddingEnd": undefined,
          "paddingHorizontal": undefined,
          "paddingLeft": undefined,
          "paddingRight": undefined,
          "paddingStart": undefined,
          "paddingTop": undefined,
          "paddingVertical": undefined,
          "position": undefined,
          "right": undefined,
          "start": undefined,
          "top": undefined,
          "width": undefined,
          "zIndex": undefined,
        },
        [
          {
            "alignContent": undefined,
            "alignItems": "center",
            "alignSelf": undefined,
            "flex": 1,
            "flexBasis": undefined,
            "flexDirection": "column",
            "flexGrow": undefined,
            "flexShrink": undefined,
            "flexWrap": undefined,
            "justifyContent": "center",
          },
          [
            undefined,
            [
              null,
              undefined,
            ],
          ],
        ],
      ]
    }
  >
    <Text
      style={
        [
          {
            "fontSize": 20,
            "fontWeight": "500",
            "letterSpacing": 0.15,
          },
          {
            "color": "#000000",
          },
          [
            undefined,
            [
              {
                "marginBottom": 4,
              },
              undefined,
            ],
          ],
        ]
      }
    >
      Männlich
    </Text>
    <View
      style={
        [
          {
            "backgroundColor": undefined,
            "borderBottomColor": undefined,
            "borderBottomEndRadius": undefined,
            "borderBottomLeftRadius": undefined,
            "borderBottomRightRadius": undefined,
            "borderBottomStartRadius": undefined,
            "borderBottomWidth": undefined,
            "borderColor": undefined,
            "borderEndColor": undefined,
            "borderEndWidth": undefined,
            "borderLeftColor": undefined,
            "borderLeftWidth": undefined,
            "borderRadius": undefined,
            "borderRightColor": undefined,
            "borderRightWidth": undefined,
            "borderStartColor": undefined,
            "borderStartWidth": undefined,
            "borderStyle": undefined,
            "borderTopColor": undefined,
            "borderTopEndRadius": undefined,
            "borderTopLeftRadius": undefined,
            "borderTopRightRadius": undefined,
            "borderTopStartRadius": undefined,
            "borderTopWidth": undefined,
            "borderWidth": undefined,
            "bottom": undefined,
            "end": undefined,
            "height": undefined,
            "left": undefined,
            "margin": undefined,
            "marginBottom": undefined,
            "marginEnd": undefined,
            "marginHorizontal": undefined,
            "marginLeft": undefined,
            "marginRight": undefined,
            "marginStart": undefined,
            "marginTop": undefined,
            "marginVertical": undefined,
            "maxHeight": undefined,
            "maxWidth": undefined,
            "minHeight": undefined,
            "minWidth": undefined,
            "opacity": undefined,
            "overflow": undefined,
            "padding": undefined,
            "paddingBottom": undefined,
            "paddingEnd": undefined,
            "paddingHorizontal": undefined,
            "paddingLeft": undefined,
            "paddingRight": undefined,
            "paddingStart": undefined,
            "paddingTop": undefined,
            "paddingVertical": undefined,
            "position": undefined,
            "right": undefined,
            "start": undefined,
            "top": undefined,
            "width": undefined,
            "zIndex": undefined,
          },
          [
            {
              "backgroundColor": "#E0E0E0",
              "height": 1,
            },
            [
              [
                undefined,
                undefined,
              ],
              [
                {
                  "marginBottom": 4,
                },
                undefined,
              ],
            ],
          ],
        ]
      }
    />
    <Text
      style={
        [
          {
            "fontSize": 20,
            "fontWeight": "500",
            "letterSpacing": 0.15,
          },
          {
            "color": "#000000",
          },
          [
            undefined,
            [
              {
                "marginBottom": 4,
              },
              undefined,
            ],
          ],
        ]
      }
    >
      super
    </Text>
    <View
      style={
        [
          {
            "backgroundColor": undefined,
            "borderBottomColor": undefined,
            "borderBottomEndRadius": undefined,
            "borderBottomLeftRadius": undefined,
            "borderBottomRightRadius": undefined,
            "borderBottomStartRadius": undefined,
            "borderBottomWidth": undefined,
            "borderColor": undefined,
            "borderEndColor": undefined,
            "borderEndWidth": undefined,
            "borderLeftColor": undefined,
            "borderLeftWidth": undefined,
            "borderRadius": undefined,
            "borderRightColor": undefined,
            "borderRightWidth": undefined,
            "borderStartColor": undefined,
            "borderStartWidth": undefined,
            "borderStyle": undefined,
            "borderTopColor": undefined,
            "borderTopEndRadius": undefined,
            "borderTopLeftRadius": undefined,
            "borderTopRightRadius": undefined,
            "borderTopStartRadius": undefined,
            "borderTopWidth": undefined,
            "borderWidth": undefined,
            "bottom": undefined,
            "end": undefined,
            "height": undefined,
            "left": undefined,
            "margin": undefined,
            "marginBottom": undefined,
            "marginEnd": undefined,
            "marginHorizontal": undefined,
            "marginLeft": undefined,
            "marginRight": undefined,
            "marginStart": undefined,
            "marginTop": undefined,
            "marginVertical": undefined,
            "maxHeight": undefined,
            "maxWidth": undefined,
            "minHeight": undefined,
            "minWidth": undefined,
            "opacity": undefined,
            "overflow": undefined,
            "padding": undefined,
            "paddingBottom": undefined,
            "paddingEnd": undefined,
            "paddingHorizontal": undefined,
            "paddingLeft": undefined,
            "paddingRight": undefined,
            "paddingStart": undefined,
            "paddingTop": undefined,
            "paddingVertical": undefined,
            "position": undefined,
            "right": undefined,
            "start": undefined,
            "top": undefined,
            "width": undefined,
            "zIndex": undefined,
          },
          [
            {
              "backgroundColor": "#E0E0E0",
              "height": 1,
            },
            [
              [
                undefined,
                undefined,
              ],
              [
                {
                  "marginBottom": 4,
                },
                undefined,
              ],
            ],
          ],
        ]
      }
    />
    <Text
      style={
        [
          {
            "fontSize": 20,
            "fontWeight": "500",
            "letterSpacing": 0.15,
          },
          {
            "color": "#000000",
          },
          [
            undefined,
            [
              {
                "marginBottom": 4,
              },
              undefined,
            ],
          ],
        ]
      }
    >
      14.4.2023
    </Text>
    <View
      style={
        [
          {
            "backgroundColor": undefined,
            "borderBottomColor": undefined,
            "borderBottomEndRadius": undefined,
            "borderBottomLeftRadius": undefined,
            "borderBottomRightRadius": undefined,
            "borderBottomStartRadius": undefined,
            "borderBottomWidth": undefined,
            "borderColor": undefined,
            "borderEndColor": undefined,
            "borderEndWidth": undefined,
            "borderLeftColor": undefined,
            "borderLeftWidth": undefined,
            "borderRadius": undefined,
            "borderRightColor": undefined,
            "borderRightWidth": undefined,
            "borderStartColor": undefined,
            "borderStartWidth": undefined,
            "borderStyle": undefined,
            "borderTopColor": undefined,
            "borderTopEndRadius": undefined,
            "borderTopLeftRadius": undefined,
            "borderTopRightRadius": undefined,
            "borderTopStartRadius": undefined,
            "borderTopWidth": undefined,
            "borderWidth": undefined,
            "bottom": undefined,
            "end": undefined,
            "height": undefined,
            "left": undefined,
            "margin": undefined,
            "marginBottom": undefined,
            "marginEnd": undefined,
            "marginHorizontal": undefined,
            "marginLeft": undefined,
            "marginRight": undefined,
            "marginStart": undefined,
            "marginTop": undefined,
            "marginVertical": undefined,
            "maxHeight": undefined,
            "maxWidth": undefined,
            "minHeight": undefined,
            "minWidth": undefined,
            "opacity": undefined,
            "overflow": undefined,
            "padding": undefined,
            "paddingBottom": undefined,
            "paddingEnd": undefined,
            "paddingHorizontal": undefined,
            "paddingLeft": undefined,
            "paddingRight": undefined,
            "paddingStart": undefined,
            "paddingTop": undefined,
            "paddingVertical": undefined,
            "position": undefined,
            "right": undefined,
            "start": undefined,
            "top": undefined,
            "width": undefined,
            "zIndex": undefined,
          },
          [
            {
              "backgroundColor": "#E0E0E0",
              "height": 1,
            },
            [
              [
                undefined,
                undefined,
              ],
              [
                {
                  "marginBottom": 4,
                },
                undefined,
              ],
            ],
          ],
        ]
      }
    />
    <Text
      style={
        [
          {
            "fontSize": 20,
            "fontWeight": "500",
            "letterSpacing": 0.15,
          },
          {
            "color": "#000000",
          },
          [
            undefined,
            [
              null,
              undefined,
            ],
          ],
        ]
      }
    >
      2
       kg
    </Text>
  </View>
</View>
`;
